test/pspm_load1_test.m

```diff
@@ -52,7 +52,8 @@ function generate_testdata(this)
           this.modelfiles{i} = mfn;
           this.dummyfiles{i} = dfn;
           fh = str2func(['pspm_', settings.first{i}]);
-          fh(model);
+          options = struct('overwrite', 1);
```

Comment:
I made some updates for the test function to force "overwriting" in any situation. This is required for test functions. This should not affect the updated model checking by this pull request...